core/src/main/java/org/apache/accumulo/core/conf/ClientProperty.java

```diff
@@ -81,6 +81,10 @@ public enum ClientProperty {
       "2.1.0", false),
   CONDITIONAL_WRITER_THREADS_MAX("conditional.writer.threads.max", "3", PropertyType.COUNT,
       "Maximum number of threads to use for writing data to tablet servers.", "2.1.0", false),
+  CONDITIONAL_WRITER_THREADS_MAX_FATE_TABLE("conditional.writer.threads.max.fate.table", "3",
```

Comment:
This should be a server property and not a client property.  In actual use it will only impact the manager process.  Also it seems like the property should have a fate prefix like `manager.fate.onditional.writer.threads`.  Then it would sort together w/ the other fate props.